src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/xfpage/customheaderlibs.html

```diff
@@ -31,8 +31,16 @@
 
 <sly data-sly-use.clientlib="/libs/granite/sightly/templates/clientlib.html"/>
 
+#if ( $frontentModule != "decoupled" ) 
 <sly data-sly-call="${symbol_dollar}{clientlib.all @ categories=['coralui3']}"/>
 <sly data-sly-call="${symbol_dollar}{clientlib.css @ categories=['${appId}.base']}"/>
+#end
+
+<sly data-sly-list="${page.htmlPageItems}">
```

Comment:
We need to also update `customfooterlibs.html`

Reply:
done